kombu/asynchronous/http/curl.py

```diff
@@ -252,14 +252,19 @@ def _setup_request(self, curl, request, buffer, headers, _pycurl=pycurl):
             setopt(meth, True)
 
         if request.method in ('POST', 'PUT'):
-            body = request.body.encode('utf-8') if request.body else b''
```

Comment:
you can rebase this so that we can merge this change

Reply:
Id close this one completely and go with the other one with Urllib3.

Shall I?